api/capi/doc/nnstreamer_capi.md

```diff
@@ -0,0 +1,315 @@
+# nnstreamer capi
+
+
+An application which require a data stream pipeline with deep learning model filter can be constructed and assembled easily with nnstreamer c-api.
+
+The main features of nnstreamer-capi include:
+
+- **Construction of data pipeline based on [gstreamer](https://gstreamer.freedesktop.org/)**
+
+  You can compose the data stream pipeline through nnstreamer-capi with various elements which included at gstreamer.
+
+- **[Single-shot](#singleshotapi) & [Pipeline](#pipelineapi) API**
+  
+  There are 2 types of nnstreamer-capi. Single-shot API is useful to get the output result of the model file without the composition of the data pipeline.
+  Otherwise, Pipeline API allows developers to manipulate and compose pipelines with various [elements](#elementapi).
```

Comment:
==>
On the other hand, Pipeline API allows developers to construct and execute pipelines with multiple neural network models, multiple input and output nodes, multiple data processors, pre-and-post processors, and various data path manipulators. Besides, if the input is online data (or streamed data), Pipeline APIs simplify your application and improve its performance.

Reply:
it's applied